src/test/java/com/cegeka/academy/web/rest/ChallengeControllerIT.java

```diff
@@ -0,0 +1,90 @@
+package com.cegeka.academy.web.rest;
+
+import com.cegeka.academy.AcademyProjectApp;
+import com.cegeka.academy.domain.Challenge;
+import com.cegeka.academy.domain.ChallengeCategory;
+import com.cegeka.academy.domain.User;
+import com.cegeka.academy.repository.ChallengeCategoryRepository;
+import com.cegeka.academy.repository.ChallengeRepository;
+import com.cegeka.academy.repository.UserRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MockMvcBuilder;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.junit.jupiter.api.*;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+import java.util.Date;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
+
+@SpringBootTest(classes = AcademyProjectApp.class)
+public class ChallengeControllerIT {
```

Comment:
Clasele pentru test, ar trebui sa contina numele Test la final nu IT

Reply:
Celelalte se terminau in IT. M-am gandit ca venea de la Implementation Test. O sa modific tho